lucida/commandcenter/app.py

```diff
@@ -43,7 +42,7 @@ def flask_listener():
                 threaded=True)
 
 def web_socket_listener():
-    print 'Start web socket at ' + str(cmd_port)
+    print 'Start web socket at 8081' 
```

Comment:
I think it would actually be better if specified on creation. Eg. in terminal if type something like "start lucida -8081".

Unless there is some advantage to storing it in the DB that you have in mind?